### PR TITLE
Improvements for table view

### PR DESCRIPTION
**Pull request by robertkowalski**
- show conflicts / conflict-count in table-view
 
to create a conflict for the document `elephant` in `animaldb` use the `new_edits=false` parameter with a random rev:

```
curl -XPUT http://localhost:8000/animaldb/elephant?new_edits=false -d '{"_rev":"4-afae890a0310210db079b0f49fb2569d","foo":"bar"}' -H 'Content-Type: application/json'
```

### CODE REVIEW

**Comment by benkeen** [comment]
I ran that command but don't see any new addition in the table view content for that DB. How new is that feature? Do I need to update couch?

**Comment by robertkowalski** [comment]
@benkeen did you enable `include_docs`?

File: app/addons/documents/index-results/index-results.components.react.jsx [hunk 1]
```diff
@@ -126,21 +126,40 @@ function (app, FauxtonAPI, React, Stores, Actions, Components, Documents, Fauxto
...
+          <div style={{display: 'inline', marginLeft: '5px'}} title={textAttachments}>
+            <i className="icon fonticon-paperclip"></i>{attachmentCount}
+          </div>
+        );
+      }
+
+      if (conflictCount) {
+        textConflicts = conflictCount === 1 ? conflictCount + ' Conflict' : conflictCount + ' Conflicts';
+        conflictIndicator = (
+          <div className="tableview-conflict" data-conflicts-indicator style={{display: 'inline'}} title={textConflicts}>
```
Comment (benkeen) [inline app/addons/documents/index-results/index-results.components.react.jsx]: Might be nice to explicitly set `data-conflicts-indicator="true"`. I didn't know `.table [data-conflicts-indicator="true"]` would match this.

**Comment by benkeen** [comment]
Nice! +1

**Comment by robertkowalski** [comment]
superseded by #670

**Pull request closed without merging**